packages/react-storage/src/components/StorageBrowser/createStorageBrowser/types.ts

```diff
@@ -225,8 +227,13 @@ export interface StorageBrowserProviderProps<V = {}>
 /**
  * @description `StorageBrowser` component, provider and view components
  */
-export interface StorageBrowserType<K = string, V = {}> {
-  (props: StorageBrowserProps<K, V>): React.JSX.Element;
+export interface StorageBrowserType<
+  LocationActionNameType = string,
```

Comment:
Named generics names should start with a captial `T` to allow for a distinct delineation between type variables and actual types. Also prefer less verbosity for easier readability, e.g. `TActionType`, `TViews`